ordered_product/tests/test_ordered_product_page.py

```diff
@@ -0,0 +1,295 @@
+import pytest
+from supplier_product.models import SupplierProduct
+from delivery_location.models import DeliveryLocation
+from ordered_product.models import OrderedProduct
+from product.models import Product
+from django.core.exceptions import ObjectDoesNotExist
+
+
+@pytest.fixture
+def valid_quantity():
+    return '5'
+
+
+@pytest.fixture
+def supplier_product0_id(supplier_product0):
+    return supplier_product0.supplier_product_id
+
+
+@pytest.fixture
+def invalid_supplier_product_id():
+    return '0'
+
+
+@pytest.fixture
+def invalid_delivery_location_id():
```

Comment:
same here

Reply:
I didn't delete these an invalid field fixtures because I'm using these fixture in some functions and I want to be clear that these fields invalid.